views/portal-report.pug

```diff
@@ -1,9 +1,9 @@
 html
   head
-    title Concord Portal Report
+    title Concord Log Puller
```

Comment:
I've renamed it so it's a bit more clear where we are. E.g. if some users report issues while trying to get logs, it could be unclear whether it was still in Portal or here.